atd-vzv/src/views/dashboard/queries/socrataQueries.js

```diff
@@ -0,0 +1,10 @@
+import { thisYear, lastYear, today, todayMonthYear } from "../helpers/time";
+
+// Year to date queries
+export const fatalitiesYTDUrl = `https://data.austintexas.gov/resource/y2wy-tgr5.json?$where=death_cnt > 0 AND crash_date between '${thisYear}-01-01T00:00:00' and '${today}T23:59:59'`;
```

Comment:
I could see us start to extract the two different endpoints to be stored as vars. 

ex: 
```js
const crashEndpointUrl = "https://data.austintexas.gov/resource/y2wy-tgr5.json"
const demographicsEndpointUrl = "https://data.austintexas.gov/resource/xecs-rpy9.json"
```

Reply:
Good call in case one of these urls changes. Just implemented this in #603.